source/installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-system-service.txt

```diff
@@ -1,3 +1,67 @@
+.. _rhel-system-service:
+
 ==========================================================
 Run Relational Migrator as a System Service on RHEL/CentOS
-==========================================================
\ No newline at end of file
+==========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+When you install Relational Migrator on a RHEL/CentOS server, you can also 
+set up Relational Migrator to run as a system service. By running Relational Migrator as 
+a system service, Relational Migrator automatically starts when the server starts.
+
+Steps
+-----
+
+1. Create a ``/etc/systemd/system/migrator.service`` file. 
+
+   Copy the text below into the ``/etc/systemd/system/migrator.service`` file. 
+   Replace the binary path as needed. 
+
+   .. code-block::
+
+      [Unit]
+      Description=MongoDB Relational Migrator
+
+      [Service]
+      ExecStart="/opt/mongodb-relational-migrator/bin/MongoDB Relational Migrator"
+
+      [Install]
+      WantedBy=multi-user.target
+
+#. Reload ``systemd`` to read the newly created service:
+
```

Comment:
(markup)
This codeblock is missing from the HTML.

Reply:
Done.